src/components/Header.tsx

```diff
@@ -32,7 +32,7 @@ const Header = () => {
           <img src={window.innerWidth < 992 ? imageSm : imageXl} alt="logo" />
         </a>
         <SearchField />
-        <a href="/favorites" className="header-nav-link">
+        <a href="/favorites/1" className="header-nav-link">
```

Comment:
what does 1 mean?

Reply:
when Dima was creating pagination and due to we need to save page by link we decided to use such a way to transfer pagination page number